deploy/charts/rook-ceph/values.yaml

```diff
@@ -275,15 +275,15 @@ csi:
   #rbdLivenessMetricsPort: 9080
   #kubeletDirPath: /var/lib/kubelet
   #cephcsi:
-    #image: quay.io/cephcsi/cephcsi:v3.4.0
+    #image: quay.io/cephcsi/cephcsi:v3.5.0
   #registrar:
-    #image: k8s.gcr.io/sig-storage/csi-node-driver-registrar:v2.3.0
+    #image: k8s.gcr.io/sig-storage/csi-node-driver-registrar:v2.4.0
   #provisioner:
-    #image: k8s.gcr.io/sig-storage/csi-provisioner:v3.0.0
+    #image: k8s.gcr.io/sig-storage/csi-provisioner:v3.1.0
   #snapshotter:
```

Comment:
deliberately kept snapshotter aside as its not updated for ceph csi release too. May be along with resizer , it can be updated. I expect the resizer to land soon.

Reply:
are we going to wait till the resizer is released to merge this PR? if not IMO as the snapshotter release is done 18 days back it can be updated as part of this PR.

Reply:
No.. we are not waiting.. but snapshotter is not tested against 3.5 release of ceph csi and there were some breaking changes in snapshotter 5.0.0 release too. so it has to be accomodated first in ceph csi to be in comfortable position here. so I would keep aside the snapshotter update till then.